.github/workflows/ci.yml

```diff
@@ -79,3 +79,26 @@ jobs:
     steps:
       - name: Check whether all jobs pass
         run: echo '${{ toJson(needs) }}' | jq -e 'all(.result == "success")'
+
+  # Publish snapshot packages. These jobs will only be triggered when the CI is executed on main.
```

Comment:
How much slower will the CI be because of this?

Reply:
So that almost doubles it, making it ~20mins. I don't see it as particularly problematic. But we can always do it later on a nightly basis if it turns out to be too long.

Reply:
So far the cron job is set to be run at 6am, I think it's ok, if not we can change it later as you say.